### PR TITLE
[BUGFIX] Calc width for proportion bars

### LINKED ISSUE #189: Incomplete display of numbers in statistics
I've there is only a small percentage of nodes with a certain attribute, but the absolute number already got 3 digits, the last digit is no longer shown.

Is this example it seems like there are only 10 nodes with the beta branch autoupdater:
image

In reallity there are 102, as you see when only these nodes are selected:
image

Same thing happens with the firmware versions and the online nodes.

Since it works fine with two digits, I am under the impression that there is am minimum size of the bars.

### PR DESCRIPTION
**Pull request by xf-**
fixes #189 - wait for approval in ticket or test with many nodes.

## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)

### CODE REVIEW

File: lib/proportions.js [hunk 1]
```diff
@@ -64,9 +64,8 @@ define(['d3-interpolate', 'snabbdom', 'filters/genericnode', 'helper'],
           var th = V.h('th', a);
           var td = V.h('td', V.h('span', {
             style: {
-              width: Math.round(v * 100) + '%',
-              backgroundColor: scale(v),
-              color: 'white'
+              width: 'calc(25px + ' + Math.round(v * 90) + '%)',
```
Comment (genofire) [inline lib/proportions.js:67]: One question:
`25px != .5em` where you got `25px` ?
https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_proportion.scss#L22

the `90` looks like found in tests to

File: lib/proportions.js [hunk 2]
```diff
@@ -64,9 +64,8 @@ define(['d3-interpolate', 'snabbdom', 'filters/genericnode', 'helper'],
           var th = V.h('th', a);
           var td = V.h('td', V.h('span', {
             style: {
-              width: Math.round(v * 100) + '%',
-              backgroundColor: scale(v),
-              color: 'white'
```
Comment (genofire) [inline lib/proportions.js:69]: Is there a reason, why you remove the font-color?
Maybe it should be store here: https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_proportion.scss#L17

Comment (xf-) [reply lib/proportions.js:69]: Is same color as body. Will be inherited.

File: lib/proportions.js (hunk 1 above)
Comment (xf-) [reply lib/proportions.js:67]: This is no padding. and max-width is around 250 px - 10% is 25px.Adding 25px to every bar by default the issue should resolve

File: lib/proportions.js (hunk 2 above)
Comment (genofire) [reply lib/proportions.js:69]: https://github.com/ffrgb/meshviewer/blob/future/scss/modules/_base.scss#L4 no there it is black ^^

File: lib/proportions.js (hunk 1 above)
Comment (genofire) [reply lib/proportions.js:67]: build it -> looks good

File: lib/proportions.js (hunk 2 above)
Comment (xf-) [reply lib/proportions.js:69]: yeah - you are right - but is css - i fix it

Comment (xf-) [reply lib/proportions.js:69]: added

**Review by genofire — APPROVED** [review]

**Pull request merged**